.github/workflows/ci.yml

```diff
@@ -5,10 +5,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [macos-latest, windows-latest, ubuntu-latest]
+        os: [macos-latest, windows-latest, ubuntu-20.04]
         language:
           [
-            {python-version: "3.8", toxenv: "py38"},
+            {python-version: "3.8.7", toxenv: "py38"},
```

Comment:
Does the new test suite pass for patch versions newer than 3.8.7, like 3.8.8?

Reply:
According to Python docs under ipaddress module, for Python 3.8, the ipaddress bug was fixed starting from 3.8.12.